Userland/Libraries/LibWeb/HTML/TrackEvent.idl

```diff
@@ -0,0 +1,16 @@
+#import <DOM/Event.idl>
+#import <HTML/VideoTrack.idl>
+
+// https://html.spec.whatwg.org/multipage/media.html#trackevent
+[Exposed=Window, UseNewAKString]
+interface TrackEvent : Event {
+    constructor(DOMString type, optional TrackEventInit eventInitDict = {});
+
+    // FIXME: Should be: (VideoTrack or AudioTrack or TextTrack)?
```

Comment:
you should use quotes in this comment! (and below) 😂

Reply:
That sounds pretty unnecessary to me, it's obvious what this is saying when looking at the IDL